src/sfizz/SIMDHelpers.h

```diff
@@ -723,12 +724,39 @@ template <>
 void copy<float, true>(absl::Span<const float> input, absl::Span<float> output) noexcept;
 
 namespace _internals {
+    // Number of elements in the table, odd for equal volume at center
+    constexpr int panSize = 4095;
+
+    // Table of pan values for the left channel, extra element for safety
+    const auto panData = []()
+    {
+        std::array<float, panSize + 1> pan;
+        int i = 0;
+
+        for (; i < panSize; ++i)
+            pan[i] = std::cos(i * (piTwo<double> / (panSize - 1)));
+
+        for (; i < static_cast<int>(pan.size()); ++i)
+            pan[i] = pan[panSize - 1];
+
+        return pan;
+    }();
+
     template <class T>
     inline void snippetPan(const T*& pan, T*& left, T*& right)
     {
-        const auto circlePan = piFour<float> * (static_cast<T>(1.0) + *pan++);
-        *left++ *= std::cos(circlePan);
-        *right++ *= std::sin(circlePan);
+        T p = ((*pan++) + T{1.0}) * T{0.5};
+        p = std::max<T>(0, std::min<T>(p, 1));
```

Comment:
You can use the clamp function in https://github.com/sfztools/sfizz/blob/efccb7ae446145376a59016781ece093aebb2a47/src/sfizz/MathHelpers.h#L129 if you want. We could make that `assert` debug only too.

Reply:
>We could make that assert debug only too.

This must ensure pan to be in range -1:+1, and once that modulation is in action, I'd rather not trust the input to live always in this domain.